src/btnFunctionality.js

```diff
@@ -0,0 +1,14 @@
+import { all } from "./index";
```

Comment:
Avoid importing and { all } try to send it as param of btnClear, now you have circular dependency, since is imported in index btnClear and btnClear imports from index, read about how bad is circular dependency in js modules.